dmutils/filters.py

```diff
@@ -66,3 +66,26 @@ def capitalize_first(maybe_text):
         return [capitalize_first(item) for item in maybe_text]
 
     return maybe_text
+
+# find repeated sequences of '\r\n\', optionally separated by a space character
+_multiple_newlines_re = re.compile(r'(\r\n ?){2,}')
+# find \r\n sequences
+_single_newline_re = re.compile(r'(\r\n)')
+
+
+@evalcontextfilter
+def preserve_line_breaks(eval_ctx, value, question_type=None):
```

Comment:
Final point: maybe we should take away `=None` here to stop people from being able to use this too easily.

Not sure where it's going to end up being used, but for both individual brief and individual service pages, we always have a `.type` hanging around.